pkgs/development/tools/toluapp/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, scons, lua }:
```

Comment:
Please issue a new PR for new packages, it's going to slow down the review process of this PR.